### PR TITLE
feat: [PROD-8406] add run templates trad labels

### PR DESCRIPTION
**Pull request by nborde-CSM**

### CODE REVIEW

File: Solution.yaml [hunk 1]
```diff
@@ -489,6 +498,9 @@ runTemplates:
       - 'events'
       - 'additional_parameters'
   - id: 'hidden test run template'
+    labels:
+      fr: 'Run template masqué sans paramètre'
```
Comment (esasova) [inline Solution.yaml:502]: IIRC we decided to use plural after _sans_ :slightly_smiling_face:

File: Solution.yaml [hunk 2]
```diff
@@ -551,8 +569,12 @@ runTemplates:
       - 'consumer to redis'
       - 'No param'
   - id: 'consume_to_api'
+    labels:
+      fr: >
+        Run template avec probe consumer pour l'API Cosmotech
```
Comment (esasova) [inline Solution.yaml:574]: Probably a typo, but why this translation doesn't have quotes ? and why there is a closing tag before ? :slightly_smiling_face:

**Review by esasova — APPROVED** [review]
just small remarks, LGTM :+1:

File: Solution.yaml (hunk 2 above)
Comment (nborde-CSM) [reply Solution.yaml:574]: In yaml, this form allow to write a string without quotes that is more convenient when there are already simple quote in the string, especially in french.

**Pull request merged**